src/lib/containers/synapse_table_functions/SynapseTableCell.tsx

```diff
@@ -88,14 +88,41 @@ export const SynapseTableCell: React.FC<SynapseTableCellProps> = ({
   // PORTALS-2095: Special case. If this is an EntityView, and we are rendering the 'name' column,
   // and we have a rowId and rowVersionNumber (should always be the case), and our entityIdToHeader map
   // contains the row Synapse ID, then auto-link.
-  if (isEntityView && columnName == 'name' && rowId && rowVersionNumber) {
+  if (
```

Comment:
Here's the change for SWC-6065

Reply:
Could this be 
(columnName == 'id' || columnName == 'name')
and just pass in the columnName as the displayTextField to EntityLink?

Reply:
Good idea!